### PR TITLE
Package post into pub dev

### PR DESCRIPTION
**Pull request by JoseAlba**
This PR is adding all the files that will be posted into [pub.dev](https://pub.dev/)

### CODE REVIEW

File: adaptive_breakpoints/example/test/widget_test.dart [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// This is a basic Flutter widget test.
```
Comment (rami-a) [inline adaptive_breakpoints/example/test/widget_test.dart]: Do you actually need to include this? Looks like the test does nothing

If you do need it, then please add the copyright header

File: adaptive_breakpoints/lib/adaptive_breakpoints_web.dart [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import 'dart:async';
```
Comment (rami-a) [inline adaptive_breakpoints/lib/adaptive_breakpoints_web.dart:5]: Copyright header

**Review by rami-a — APPROVED** [review]
LGTM

File: adaptive_breakpoints/example/test/widget_test.dart (hunk 1 above)
Comment (JoseAlba) [reply adaptive_breakpoints/example/test/widget_test.dart]: Done

File: adaptive_breakpoints/lib/adaptive_breakpoints_web.dart (hunk 2 above)
Comment (JoseAlba) [reply adaptive_breakpoints/lib/adaptive_breakpoints_web.dart:5]: Done

**Pull request closed without merging**